strategies.py

```diff
@@ -127,13 +128,13 @@ def play_move(self, c):
         '''
         Notable side effects:
           - finalizes the probability distribution according to
-          this roots visit counts into the class' running tally, `searches`
+          this roots visit counts into the class' running tally, `searches_pi`
           - Makes the node associated with this move the root, for future
             `inject_noise` calls.
         '''
         if not self.two_player_mode:
             self.searches_pi.append(
-                self.root.children_as_pi(self.root.position.n > self.temp_threshold))
+                self.root.children_as_pi(self.root.position.n < self.temp_threshold))
```

Comment:
lol.

Reply:
this is actually correct -- we changed stretch to squash.